bin/cron/create_rollup_tables

```diff
@@ -86,6 +86,32 @@ def main
     end
   end
 
+  Level.find_each do |level|
+    next unless level.script_levels.map(&:script).select(&:k5_draft_course?)
+
+    level.contained_levels.each do |contained_level|
+      contained_question = get_contained_question(contained_level)
+      contained_levels << {
+        level_id: level.id,
+        contained_level_id: contained_level.id,
+        contained_level_type: contained_level.type,
+        contained_level_page: page_index,
```

Comment:
Nice catch, the `page_index` should be `1` and the `position_index` should be the `contained_level` index. Done.